scripts/start

```diff
@@ -46,6 +46,10 @@ export DEVICE_HOST="http://"${DEVICE_IP:-"$DEVICE_HOSTNAME".local}""
 
 cd "$UMBREL_ROOT"
 
+echo "Setting dbcache size"
+DBCACHE_SIZE=$(awk '/MemTotal/{printf "%d\n", $2/2^10 * 0.5}'  /proc/meminfo)
```

Comment:
This sets `DBCACHE_SIZE` to 50% of RAM. It should be 50%-300 to account for the 300MB allocated to `maxmempool.`

```suggestion
DBCACHE_SIZE=$(awk '/MemTotal/{printf "%d\n", ($2/2^10 * 0.5) - 300}' /proc/meminfo)
```